web/src/components/LegalStructureDropDown.tsx

```diff
@@ -44,6 +44,7 @@ export const LegalStructureDropDown = <T,>(props: Props<T>): ReactElement => {
       setProfileData({
         ...state.profileData,
         legalStructureId: event.target.value,
+        isNonProfit: (event.target.value as string) === "nonprofit",
```

Comment:
dust - let's create a new variable specific to `isNonProfitFromFunding` so it can be removed from legal structure field and not cause confusion with intent of legal structure field.